vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/DragAndDropGridIT.java

```diff
@@ -24,6 +25,12 @@
 import com.vaadin.flow.component.grid.testbench.GridElement;
 import com.vaadin.tests.AbstractComponentIT;
 import com.vaadin.flow.testutil.TestPath;
+import org.openqa.selenium.logging.LogEntry;
+import org.openqa.selenium.logging.LogType;
+
+import java.util.List;
+import java.util.logging.Level;
+import java.util.stream.Collectors;
```

Comment:
A few unused imports here, will probably not require any new imports if the other suggestion is applied.

Reply:
Removed.